shm/src/coordinates.cpp

```diff
@@ -0,0 +1,6 @@
+#include "inc/coordinates.hpp"

```

Comment:
When PR with CMake will be merged, there won't be need to add `inc` prefix to `#include` I think.

Reply:
Yup. We can wait with that until the CMake is integrated with master. Then I can change to whatever it will be needed